rdflib/compare.py

```diff
@@ -290,7 +292,7 @@ def _initial_color(self):
 
     def _individuate(self, color, individual):
         new_color = list(color.color)
-        new_color.append((len(color.nodes)))
+        new_color.append(tuple([len(color.nodes)]))
```

Comment:
@wwaites: well, the double bracket removal is one of the tripping stones in python, so i can somehow understand why some people prefer `(tuple([...]))` instead of `((...,))`. Without seeing it as much of a problem i still changed it in b8df01f9bc53ccd01d61953108c30afae0f4b36e ;)